lib/inc/puppetdb-cli/puppetdb-cli.hpp

```diff
@@ -109,7 +109,8 @@ LIBPUPPETDB_EXPORT PuppetDBConn get_puppetdb(const std::string& config_path,
  * @return This function does not return anything.
  */
 LIBPUPPETDB_EXPORT void pdb_query(const PuppetDBConn& conn,
-                                  const std::string& query);
+                                  const std::string& query,
+                                  const bool& is_pretty);
```

Comment:
Need to add another `@param` for doxygen above https://github.com/puppetlabs/puppetdb-cli/pull/26/files#diff-98d0421500d77fa902e8b6c129990fc5R109